test/api/subgroup/create.test.js

```diff
@@ -34,8 +34,23 @@ test('tribes.subtribe.create', t => {
         t.equal(child, subgroupId, 'link/group-subgroup child')
         t.deepEqual(recps, [groupId], 'link/group-subgroup recps')
 
-        server.close()
-        t.end()
+        server.tribes.get(subgroupId, (err, group) => {
+          t.error(err, 'get subgroup')
+
+          t.deepEqual(
+            group,
+            {
+              key: subgroupKey,
+              root: groupInitMsg.key,
+              scheme: 'envelope-large-symmetric-group',
+              parentGroupId: groupId
```

Comment:
if we do end up looking up links generally then we may as well have parentGroupId + subrgroupIds:

Reply:
can do later

Reply:
Yeah i choose later haha :)